<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <hintingpreference>PreferNoHinting</hintingpreference>
    <fontweight>DemiBold</fontweight>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#LoginWidget {
	background-image: url(&quot;:img/login.png&quot;);
	background-repeat: no-repeat;
}
QCheckBox {
	color: #392518;
}

QLabel#l_remember_me {
	color: #453327;
}

QLabel#l_story_awaits {
	color: #453327;
}

QLabel#l_sign_up {
	color: #FF441C;
}

QLabel#l_error {
	color: white;
	background-color: rgb(255, 68, 28);
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
}

QLabel#l_error_user {
	border-width: 2px;
	border-style: solid;
    border-radius: 3px;
    border-color: rgb(255, 68, 28);
}

QLabel#l_error_pass {
	border-width: 2px;
	border-style: solid;
    border-radius: 3px;
    border-color: rgb(255, 68, 28);
}

QLineEdit {
	color:#392518;
	background-color: transparent;
 	padding: 0px 23px;
	border: none;
	placeholder-text-color: #392518;
}
QLabel {
	color: #4F1717;
}


QCheckBox#check_remember::indicator:unchecked {
	image: url(&quot;:/img/btn/checkbox.png&quot;)
}

QCheckBox#check_remember::indicator:checked {
	image: url(&quot;:/img/btn/checkbox_checked.png&quot;)
}

</string>
  </property>
  <widget class="QLabel" name="l_story_awaits">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>218</y>
     <width>135</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>12</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Your Story awaits!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>45</y>
     <width>112</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font: 25px EurostileBla;
     color: white;
    border-style: solid;
    border-width: 1px;
    border-radius: 4px;
    border-color: rgb(47, 180, 224);
    background-color: qlineargradient(spread:pad, x1:0.503, y1:0.704546, x2:0.503, y2:0.03, stop:0 rgba(9, 171, 224, 255), stop:0.999893 rgba(125, 220, 247, 255), stop:0.999906 rgba(222, 248, 255, 255), stop:1 rgba(222, 248, 255, 255));
}

QPushButton:hover{
    font: 25px EurostileBla;
     color: white;
    border-style: solid;
    border-width: 1px;
    border-radius: 4px;
    border-color: rgb(47, 180, 224);
    background-color: qlineargradient(spread:pad, x1:0.503, y1:0.704546, x2:0.503, y2:0.03, stop:0 rgba(4, 197, 237, 255), stop:0.999893 rgba(160, 237, 252, 255), stop:0.999906 rgba(222, 248, 255, 255), stop:1 rgba(222, 248, 255, 255));
}

QPushButton:pressed{
    font: 25px EurostileBla;
     color: white;
    border-style: solid;
    border-width: 1px;
    border-radius: 4px;
    border-color: rgb(47, 180, 224);
    background-color: qlineargradient(spread:pad, x1:0.503, y1:0.985, x2:0.503, y2:0, stop:0 rgba(222, 248, 255, 255), stop:0.01 rgba(222, 248, 255, 255), stop:0.011 rgba(67, 191, 232, 255), stop:0.35 rgba(35, 183, 230, 255), stop:0.95 rgba(92, 210, 244, 255), stop:1 rgba(71, 174, 208, 255));
}

QPushButton:disabled {
    font: 25px EurostileBla;
    color: rgb(155, 145, 133);
    border-style: solid;
    border-width: 1px;
    border-radius: 4px;
    border-color: rgb(146, 121, 103);
    
    background-color: qlineargradient(spread:pad, x1:0.503, y1:0.704546, x2:0.503, y2:0, stop:0 rgba(185, 179, 167, 255), stop:0.830508 rgba(168, 150, 137, 255), stop:1 rgba(143, 120, 110, 255));
}
</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
   <property name="iconSize">
    <size>
     <width>112</width>
     <height>111</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="l_sign_up">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>215</y>
     <width>65</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EurostileBla</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
     <underline>true</underline>
     <hintingpreference>PreferNoHinting</hintingpreference>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://storyofalicia.com&quot;&gt;&lt;span style=&quot; font-size:18px; color:#FF441C;&quot;&gt;Sign Up&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_username">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>45</y>
     <width>304</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NotEurostile</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <hintingpreference>PreferNoHinting</hintingpreference>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_remember">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>188</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
     <hintingpreference>PreferDefaultHinting</hintingpreference>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>23</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>109</y>
     <width>304</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NotEurostile</family>
     <pointsize>12</pointsize>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>Black</fontweight>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_remember_me">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>189</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>13</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Remember me</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_error">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>24</y>
     <width>272</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>9</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>ExtraBold</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Invalid username or password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_error_user">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>45</y>
     <width>304</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_error_pass">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>109</y>
     <width>304</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>l_error</zorder>
  <zorder>l_story_awaits</zorder>
  <zorder>btn_login</zorder>
  <zorder>l_sign_up</zorder>
  <zorder>input_username</zorder>
  <zorder>check_remember</zorder>
  <zorder>input_password</zorder>
  <zorder>l_remember_me</zorder>
  <zorder>l_error_user</zorder>
  <zorder>l_error_pass</zorder>
 </widget>
 <tabstops>
  <tabstop>input_username</tabstop>
  <tabstop>input_password</tabstop>
  <tabstop>check_remember</tabstop>
  <tabstop>btn_login</tabstop>
 </tabstops>
 <resources>
  <include location="master.qrc"/>
 </resources>
 <connections/>
</ui>
